docs/How-to-Create-and-Maintain-a-Tap.md

```diff
@@ -10,23 +10,57 @@ last_review_date: "1970-01-01"
 
 A tap is usually a Git repository available online, but you can use anything as long as it’s a protocol that Git understands, or even just a directory with files in it. If hosted on GitHub, we recommend that the repository’s name start with `homebrew-` so the short `brew tap` command can be used. See the [`brew` manual page](Manpage.md) for more information on repository naming.
 
-The `brew tap-new` command can be used to create a new tap along with some template files.
+The `brew tap-new` command can be used to create a new tap along with some template files:
+
+```console
+$ brew tap-new $YOUR_GITHUB_USERNAME/homebrew-tap
+Initialized empty Git repository in /opt/homebrew/Library/Taps/$YOUR_GITHUB_USERNAME/homebrew-tap/.git/
+...
+==> Created $YOUR_GITHUB_USERNAME/tap
+/opt/homebrew/Library/Taps/$YOUR_GITHUB_USERNAME/homebrew-tap
+```
+
+Next, you can push it to a new GitHub repository:
+
+```console
+$ gh repo create $YOUR_GITHUB_USERNAME/homebrew-tap --push --public --source "$(brew --repository $YOUR_GITHUB_USERNAME/homebrew-tap)"
```

Comment:
Am I supposed to do this from inside the newly-created tap directory? Is it meant to be my working directory? Or should I maintain the code elsewhere? (Feels extremely weird `cd`'ing into /opt for this purpose)